.changeset/plenty-doors-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+"functions-subscription": patch
+---
+
+[IOPLT-718] Get subscriptions by trial id and user id
```

Comment:
**suggestion**: improve the changelog

I think it's an old changelog, since there is `subscriptions`, plural

Reply:
Resolved in [7bf2b52](https://github.com/pagopa/trial-system/pull/256/commits/7bf2b52679cd429fbead9cbbbc6d8fdbbd2c1a6e)